controllers/householdController.js

```diff
@@ -0,0 +1,70 @@
+// Get models & utils
+const db = require("../models");
+const { validateObjectId } = require("./utils");
+const { ObjectId } = require("mongoose").Types // Import mongoose ObjectId for validation, etc
+
+// Define some custom errors
+class ValidationError extends Error {};
+ValidationError.prototype.message = "BADNAME";
+
+// Define methods for manipulation of Households collection
+module.exports = {
+
+  // New household from scratch
+  create: function (name) {
+
+    // Return promise of function since it needs to do some asynch stuff
+    return new Promise((resolve, reject) => {
+
+      // Validation checks
+      if (typeof (name) !== 'string') {
+        reject(new ValidationError("Household name must be a string. "));
+      }
+      // TODO: The following validation code results in triggering the if statement when it should not and I can't figure out why right now. Not really needed for MVP though.
+      // if (name === '[object Undefined]' || name === undefined || name === null || isNaN(name)) {
+      //   reject(new ValidationError(`"${name}" is not a valid value for Household name.`));
+      // }
+      if (name.length < 3) {
```

Comment:
Is this looking at the string length? I think a 3 letter last name is legitimate. Maybe 1 letter or 2?

Reply:
Will Change to 2